src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,116 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './TodoList.scss';
+import classnames from 'classnames';
+import { getTodos } from '../../api';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
+type Props = {
+  addUserId: (userId: number) => void,
+};
+
+export const TodoList: React.FC<Props> = ({
+  addUserId,
+}) => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [input, setInput] = useState('');
+  const [option, setOption] = useState('all');
+
+  useEffect(() => {
+    getTodos().then(allTodos => setTodos(allTodos));
```

Comment:
you should handle errors here, if getTodos returns an error you should catch the error here